invenio_rdm_records/resources/config.py

```diff
@@ -91,6 +91,8 @@ class RDMRecordResourceConfig(RecordResourceConfig, ConfiguratorMixin):
     routes["item-review"] = "/<pid_value>/draft/review"
     routes["item-actions-review"] = "/<pid_value>/draft/actions/submit-review"
     # Community records
+    routes["item-remove-community"] = "/<pid_value>/communities"
+    routes["item-remove-community-list"] = "/<pid_value>/communities/<community_id>"
```

Comment:
move to an `/actions` endpoint, something similar to what Zenodo does:

```shell
POST /communities/<community_id>/curaterecord/

Payload:

{
    "recid": "1133576",
    "action":"remove"
}
```

This would enable support for the community addition, without having to send a "PUT" operation on the record. Therefore, we would be treating communities in a similar way we are treating PIDs.